cubeb-backend/src/capi.rs

```diff
@@ -139,7 +139,7 @@ pub unsafe extern "C" fn capi_stream_init<CTX: Context>(
     state_callback: ffi::cubeb_state_callback,
     user_ptr: *mut c_void,
 ) -> c_int {
-    let mut ctx = &mut *(c as *mut CTX);
```

Comment:
(commenting here since you can't comment on the commit message):

"change the signature of to take &self" is missing a word